packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/nodejs/sdkforyourapp.md

```diff
@@ -1,41 +1,12 @@
-## TO DO for NodeJS
 
-#### Step 1: Set up your app for .Net 4.8 or greater
+#### Step 1: Install the SDK into your project application
 
-The SDK and samples are built using .Net 4.8.
+Download and install the SDK into your Express application. See [Building the SDK](https://github.com/okta/okta-auth-js#building-the-sdk) for further information.
```

Comment:
The build step won't be needed if the devs are using the SDK in their own app. 

The steps should be:
Firstly, install okta-auth-js, `npm install @okta/okta-auth-js`
Then explain how to use the SDK methods. Details can be found [here](https://github.com/okta/okta-auth-js/blob/master/docs/idx.md)

Reply:
Thanks, I recast this page with these updates.